templates/authenticate/form.php

```diff
@@ -1,6 +1,6 @@
 <?php /** @var stdClass $data */ ?>
 
-<form method="post" action="<?php echo esc_url( $data->form_url ); ?>">
+<form method="GET" action="<?php echo esc_url( $data->form_url ); ?>">
```

Comment:
Would there be a way to not have to change this to `GET`, maybe by transforming the request on the server side from POST to GET (before the call to `handleAuthorizeRequest`)?

I'm asking this because `form.php` is part of our "public API", in the sense that users might have overridden this template, so they wouldn't get this change. I realise no one is probably using this library yet, but I think the principle still applies.

In short we're changing our "public API" because of a bug in a library we're using internally. I think It would be best if the template would not be dependent on the fact we're using the bshaffer library, as we might stop using it in the future, or the bug might get fixed.

Reply:
I understand your concern and based on Alex's suggestion I have copied over the value with explanation in the comment.